pkg/autoscaler/aggregation/bucketing.go

```diff
@@ -44,12 +44,7 @@ func (t *TimedFloat64Buckets) Record(time time.Time, name string, value float64)
 	defer t.bucketsMutex.Unlock()
 
 	bucketKey := time.Truncate(t.granularity)
-	bucket, ok := t.buckets[bucketKey]
-	if !ok {
-		bucket = float64Bucket{}
-		t.buckets[bucketKey] = bucket
-	}
-	bucket.record(name, value)
+	t.buckets[bucketKey] += value
```

Comment:
Maybe we can add a little bit of prose to this line or this method saying something like:

> Metrics received in a certain timeframe are all summed up. This assumes that we don't take multiple readings of the same metric in the same bucket (per second currently). The only case where this might happen currently is if an activator scales from 0. The metrics for that bucket might be off by exactly "1" as that poke always reports a concurrency of 1. Since we're windowing metrics anyway, that slight skew is acceptable.

Just to document the train of thought for future changes.

Reply:
SG.